src/PolyhedralGeometry/PolyhedralFan/properties.jl

```diff
@@ -143,7 +143,7 @@ julia> n_maximal_cones(PF)
 2
 ```
 """
-n_maximal_cones(PF::PolyhedralFan) = pm_object(PF).N_MAXIMAL_CONES
+n_maximal_cones(PF::_FanLikeType) = pm_object(PF).N_MAXIMAL_CONES
```

Comment:
Also here and in other places (again: feel free to ignore for this PR, just would be nice to have this *eventually*
```suggestion
n_maximal_cones(PF::_FanLikeType) = pm_object(PF).N_MAXIMAL_CONES::Int
```